native/src/renderer/windowed.rs

```diff
@@ -2,10 +2,24 @@ use crate::MouseCursor;
 
 use raw_window_handle::HasRawWindowHandle;
 
+use crate::Color;
+#[derive(Debug)]
+pub struct Style {
+    pub window_background : Color /*= Color::WHITE*/,
+    pub text_color : Color /*= Color::BLACK*/,
+    pub text_size: f32 /*= 20.0*/,
+}
```

Comment:
This is similar to what I had in mind to configure renderers. The plan, however, is to make each renderer define its own configuration!